h/services/user_delete.py

```diff
@@ -168,14 +180,17 @@ def delete_annotations(self, user):
         )
 
         # Whenever we update annotations we also need to update the corresponding annotation_slims.
-        num_deleted_annotation_slims = self.db.execute(
+        deleted_annotation_slim_ids = self.db.scalars(
             update(AnnotationSlim)
             .where(AnnotationSlim.pubid.in_(deleted_annotation_ids))
             .values({"deleted": True, "updated": now})
-        ).rowcount
-        if num_deleted_annotation_slims:
+            .returning(AnnotationSlim.id)
+        ).all()
+        if deleted_annotation_slim_ids:
             log.info(
-                "Updated %d rows from annotation_slim", num_deleted_annotation_slims
+                "Marked annotation_slims belonging to %r as deleted: %s",
+                user,
+                ", ".join(str(id_) for id_ in sorted(deleted_annotation_slim_ids)),
```

Comment:
Log the actual ids of the annotation_slims being deleted as well. `sorted()` is mainly to enable test assertions to work without being flakey, although making the code more predictable (always logging ids in alphabetical order) may not be a bad thing for the logs as well.